Algorithms.Tests/Numeric/GaussJordanEliminationTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Algorithms.Numeric;
+using NUnit.Framework;
+
+namespace Algorithms.Tests.Numeric
+{
+    /// <summary>
+    /// Class for testing Gauss-Jordan Elimination Algorithm.
+    /// </summary>
+    public static class GaussJordanEliminationTests
+    {
+        [Test]
+        public static void SolveMatrixThrowsException()
```

Comment:
From the name of the test it's not clear **why** solver should fail, what is wrong with the input.